ctlcmd/drscmd.py

```diff
@@ -157,6 +157,8 @@ def add_args(self):
 
   def run(self, args):
     ## First line in file contains convertion information
+
+    ##TODO: decide what to do with this vs the root file, add to root file or keep with txt file?
```

Comment:
Leave this file for now. Keep the TODO tag so that we can keep track of what needs to be done.